lighthouse-core/lib/log.js

```diff
@@ -16,9 +16,39 @@
  */
 'use strict';
 
-const npmlog = require('npmlog');
+const debug = require('debug');
 
-// Use `npmlog` in node, `console` in browser and electron.
-const isNode = !!process && !process.browser && !process.versions.electron;
+function setLevel(level) {
+  if (level === 'verbose') {
+    debug.enable('*');
+  } else if (level === 'error') {
+    debug.enable('*:error');
+  } else {
+    debug.enable('*, -*:verbose');
+  }
+}
 
-module.exports = isNode ? npmlog : console;
+let loggers = {};
+function _log(title, logargs) {
+  const args = Array.from(logargs).slice(1);
+  if (!loggers[title]) {
+    loggers[title] = debug(title);
+  }
+  return loggers[title].apply(null, args);
+}
+
+module.exports = {
+  setLevel,
+  log: function(title) {
+    return _log(title, arguments);
```

Comment:
ugh, we really need to upgrade to node 6 and rest parameters
```
log: function(title, ...args) {
  return _log(title, args);
}
```
and avoid all the slice business above